src/com/palmergames/bukkit/towny/object/Translation.java

```diff
@@ -357,4 +357,26 @@ public static void addTranslations(Map<String, Map<String, String>> addedTransla
 					}
 		}
 	}
+	
+	private static String getTranslationValue(Map.Entry<String, Object> entry) {
+		// Messages blocked from being overriden.
+		if (entry.getKey().toLowerCase().startsWith("msg_ptw_warning")) {
+			// Get the defaultLocale's translation of the PTW warnings.
+			String msg = String.valueOf(translations.get(defaultLocale).get(entry.getKey()));
```

Comment:
```suggestion
			String msg = String.valueOf(entry.getValue());
```

Reply:
I think this is required in order for any of the translated messages to be used instead of the default locale only.